Dockerfile.development

```
@@ -1,4 +1,5 @@
-FROM dependabot/dependabot-core
+ARG FROM=dependabot/dependabot-core
+FROM $FROM
```

args and keys with the same name are so easily confusing... I'd prefer to call this something slightly more verbose to reduce confusion/mental overhead... open to ideas, or just do

```suggestion
ARG FROM_IMAGE=dependabot/dependabot-core
FROM $FROM_IMAGE
```